pkg/oc/cli/admin/policy/reconcile_sccs.go

```diff
@@ -10,25 +10,30 @@ import (
 	sccutil "github.com/openshift/origin/pkg/security/securitycontextconstraints/util"
 	kapierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/sets"
-	kapi "k8s.io/kubernetes/pkg/apis/core"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	kapihelper "k8s.io/kubernetes/pkg/apis/core/helper"
-	kcoreclient "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/core/internalversion"
 	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/genericclioptions"
+	"k8s.io/kubernetes/pkg/kubectl/genericclioptions/printers"
+	"k8s.io/kubernetes/pkg/kubectl/scheme"
 
+	securityv1 "github.com/openshift/api/security/v1"
+	securityv1client "github.com/openshift/client-go/security/clientset/versioned/typed/security/v1"
```

Comment:
`securityv1typedclient`

Reply:
Do we do this as well for the `corev1client`?